doc/reference/javascript_reference.rst

```diff
@@ -1213,6 +1213,10 @@ The notification system in Odoo is designed with the following components:
   destroy notifications whenever a request is done (with a custom_event). Note
   that the web client is a service provider.
 
+- a client action *display_notification*: this allow to call directly the
```

Comment:
this allows to trigger the display of a notification from python (e.g. in the method called when the user clicked on a button of type object).